clinica/iotools/bids_utils.py

```diff
@@ -85,117 +96,86 @@ def create_participants_df(
     import numpy as np
 
     from clinica.iotools.converters.adni_to_bids.adni_utils import load_clinical_csv
-    from clinica.utils.stream import cprint
 
-    fields_bids = ["participant_id"]
-    prev_location = ""
-    index_to_drop = []
-    subjects_to_drop = []
```

Comment:
I would also make sure `study_name` is a supported value here. Something like:

```python
if study_name not in (StudyName.ADNI, StudyName.OASIS, StudyName.OASIS3):
    raise ValueError(f"Study {study_name.value} is not supported by 'create_participants_df'.")
```

And it'd be great to add a unit test covering this error.
If we adapt this function for other studies later, we'll update these.
WDYT ?